test/mocha.opts

```diff
@@ -2,3 +2,4 @@
 --compilers js:babel-register,jsx:babel-register
 --require airbnb-js-shims
 --recursive
+test/_helpers test/**/*_spec*
```

Comment:
This file needs a trailing newline; separately, the spec dir shouldn’t go here - i should be able to run `mocha path/to/file` and *not* run every test (and I’d prefer ever file inside “test” get treated as a spec, not just ones following a naming convention)